code/obj/machinery/computer/chem_requester.dm

```diff
@@ -0,0 +1,152 @@
+var/list/datum/chem_request/chem_requests = list()
```

Comment:
i'd like the file moved to the chemistry module folder, other than that all looks good